CI fails due to VTK CMake error

Since a few days we get

```
CMake Error at /usr/lib64/cmake/vtk/VTK-targets.cmake:553 (set_target_properties):
  The link interface of target "VTK::nlohmannjson" contains:
    nlohmann_json::nlohmann_json
  but the target was not found.  Possible reasons include:
    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
```

in our Fedora 40 based CI pipelines. As can be seen in #131, this is not fixed by updating to Fedora 41.

More details: https://discourse.vtk.org/t/cmake-error-vtk-nlohmannjson/15170

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -205,6 +205,7 @@ if(NOT DEFINED KASPER_USE_VTK)
     find_package(GLEW QUIET)
     find_package(Freetype QUIET)
     find_package(JsonCpp QUIET)
+    find_package(nlohmann_json QUIET) # Workaround for https://gitlab.kitware.com/vtk/vtk/-/issues/19588
     if(VTK_FOUND)
         message("VTK was found and enabled automatically. Set KASPER_USE_VTK=OFF to change this.")
     endif()
@@ -214,6 +215,7 @@ elseif(KASPER_USE_VTK)
     find_package(GLEW QUIET)
     find_package(Freetype QUIET)
     find_package(JsonCpp QUIET)
+    find_package(nlohmann_json QUIET) # Workaround for https://gitlab.kitware.com/vtk/vtk/-/issues/19588
 endif()
 option( KASPER_USE_VTK "Build VTK dependent modules" ${VTK_FOUND} )
 

